packages/accessibility/src/behaviors/SplitButton/splitButtonBehavior.ts

```diff
@@ -25,6 +26,8 @@ const splitButtonBehavior: Accessibility = props => {
             keyCombinations: [
               { keyCode: keyboardKey.Escape },
               { keyCode: keyboardKey.ArrowUp, altKey: true },
+              { keyCode: keyboardKey.Tab, shiftKey: false },
```

Comment:
Are we absolutely sure that the items inside the menu cannot have elements that can be focused?

Reply:
yes, menu can only contain menu items, nothing else.